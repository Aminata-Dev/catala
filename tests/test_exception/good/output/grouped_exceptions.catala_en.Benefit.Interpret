[RESULT] Computation successful! Results:
[RESULT] benefit = $2000.00
[RESULT] person = Person {"age": 26, "disabled": true}
