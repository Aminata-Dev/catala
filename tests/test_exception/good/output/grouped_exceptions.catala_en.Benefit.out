[RESULT] Computation successful! Results:
[RESULT] benefit = $2000
